core/migrations/8_deploy_store.js

```diff
@@ -9,7 +9,19 @@ module.exports = async function(deployer, network, accounts) {
   const keys = getKeysForNetwork(network, accounts);
   const controllableTiming = enableControllableTiming(network);
 
-  const { contract: store } = await deploy(deployer, network, Store, Timer.address, { from: keys.deployer });
+  // Initialize both fees to 0.
+  const initialFixedOracleFeePerSecondPerPfc = { rawValue: "0" };
```

Comment:
nit: I noticed that Store.js does not have a unit test for the constructor. Is it worth adding a small one there to check that state variables are set correctly just for peace of mind?

Reply:
SG, will add a constructor test.

Reply:
Done.